.devcontainer/post-create.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+PYTHON_UTILS=("yamllint" "pre-commit")
+GITHUB_UTILS=("")
+GOLANG_UTILS=("github.com/google/yamlfmt/cmd/yamlfmt@latest" "github.com/goreleaser/goreleaser@latest")
+APT_UTILS=("shellcheck" "vim")
+NODE_UTILS=("@commitlint/cli" "@commitlint/config-conventional")
+set -e
+
+# Install Python tools
+if [[ $(python --version) != "" ]]; then
+    echo ====================================================
+    echo Installing Python tools...
+    export PYTHONUSERBASE=/tmp/pip-tmp
+    export PIP_CACHE_DIR=/tmp/pip-tmp/cache
+    PIPX_DIR=""
+    if ! type pipx >/dev/null 2>&1; then
+        pip3 install --disable-pip-version-check --no-cache-dir --user pipx 2>&1
+        /tmp/pip-tmp/bin/pipx install --pip-args=--no-cache-dir pipx
+        PIPX_DIR="/tmp/pip-tmp/bin/"
+    fi
+    for util in "${PYTHON_UTILS[@]}"; do
+        if ! type "${util}" >/dev/null 2>&1; then
+            "${PIPX_DIR}pipx" install --system-site-packages --pip-args '--no-cache-dir --force-reinstall' "${util}"
+        else
+            echo "${util} already installed. Skipping."
+        fi
+    done
+    rm -rf /tmp/pip-tmp
+fi
+
+# Install tools
+echo ====================================================
+echo "Installing tools from Github..."
+for util in "${GITHUB_UTILS[@]}"; do
+    if ! type "${util}" >/dev/null 2>&1; then
+        curl -s "https://raw.githubusercontent.com/${util}" | bash
+        echo ""
+    else
+        echo "${util} already installed. Skipping."
+    fi
+done
+
+# Install Golang tools
+echo ====================================================
+echo Installing Golang tools...
+for util in "${GOLANG_UTILS[@]}"; do
+    if ! type "${util}" >/dev/null 2>&1; then
+        go install "${util}"
+    else
+        echo "${util} already installed. Skipping."
+    fi
+done
+
+# Install Node tools
```

Comment:
Does this script need the `node` tools?

Reply:
yes, I'm using it to install @commitlint/cli commitlint/config-conventional